src/OrchardCore.Modules/OrchardCore.Users.TimeZone/Services/IUserTimeZoneService.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
+using OrchardCore.Modules;
+using OrchardCore.Users.TimeZone.Models;
+
+namespace OrchardCore.Users.TimeZone.Services
+{
+    public interface IUserTimeZoneService
```

Comment:
It's an interface in it's own module used only by this module. Hence you don't need an interface. Just create a class if you need to separate the logic from where you call it (which I doubt too) and register/resolve the class directly.

Reply:
I kept it because I mainly reproced the pattern of Themes which are also having an ISiteTheService.cs and IThemeService.cs in the same module and they are not used anywhere else than in that module. I was asking myself why it was needed and did not found why so I kept the pattern.